docs-chef-io/content/inspec/attestations.md

```diff
@@ -0,0 +1,95 @@
++++
+title = "Attestations"
+draft = false
+gh_repo = "inspec"
+
+[menu]
+  [menu.inspec]
+    title = "Attestations"
+    identifier = "inspec/reference/attestations.md Attestations"
+    parent = "inspec/reference"
+    weight = 140
++++
+
+Attestation is a mechanism to mark `Not Reviewed (N/R)` controls to `passed` or `failed` manually using an attestation file.An attestation file identifies:
+
```

Comment:
We need a real-life example of when you would use an attestation. "Testing the fire alarms" is a good example.